README.md

```diff
@@ -1,9 +1,17 @@
 # polder-schema.org-template
 
-The goal of POLDER is to facilitate single-window metadata search in support of polar science and research communities. POLDER is encouraging data holders to generate schema.org metadata descriptions to support this goal. The polder-schema.org-template repository on github has been created to foster community agreement on [documentation](https://github.com/POLDER-Crew/polder-schema.org-template/tree/master/Documentation) and [examples](https://github.com/POLDER-Crew/polder-schema.org-template/tree/master/Examples) for implementing schema.org with a focus on the polar community search needs. Wherever possible, our examples and guidance documents will draw from the [ESIP Science on Schema.org guidelines](https://github.com/ESIPFed/science-on-schema.org).
+The goal of POLDER is to facilitate single-window metadata search in support of polar science and research communities. POLDER is encouraging data holders to generate schema.org metadata descriptions to support this goal. The Best-Practices-schema.org repository on github has been created to foster community agreement on [documentation](https://github.com/POLDER-Crew/polder-schema.org-template/tree/master/Documentation) and [examples](https://github.com/POLDER-Crew/polder-schema.org-template/tree/master/Examples) for implementing schema.org with a focus on the polar community search needs. Wherever possible, our examples and guidance documents will draw from the [ESIP Science on Schema.org guidelines](https://github.com/ESIPFed/science-on-schema.org).
```

Comment:
I think you might need to update the links in here to if you changed the name of this repo